akka-docs/src/main/paradox/remoting-artery.md

```diff
@@ -864,3 +864,11 @@ spec:
 There is currently no way to limit the size of a memory empty dir but there is a [pull request](https://github.com/kubernetes/kubernetes/pull/63641) for adding it.
 
 Any space used in the mount will count towards your container's memory usage.
+
+
+### Flight Recorder
+
+When running on JDK 11 Artery specific flight recording is available through the [Java Flight Recorder (JFR)](https://openjdk.java.net/jeps/328).
+
```

Comment:
would be nice with some basic instructions of JVM flags that are needed to enable it, or link to JFR docs

Reply:
That's mentioned in that JEPS link, I don't think I have seen much better docs anywhere. I thought since this is a extra-power user feature so far, our custom events is probably mostly useful to use debugging remoting, I didn't want to make it too inviting.